app/lib/gilded_rose.rb

```diff
@@ -0,0 +1,24 @@
+# => This class update the value of store 
+
+require_relative 'item'
+require_relative 'utility_functions'
+class GildedRose
+  attr_reader :items
+
+  def initialize(items)
+    @items = items
+  end
+
+  def update_quality()
+    @items.each do |item|
```

Comment:
since you're declaring an `attr_reader` for `items`, it's not necessary access the variable directly, you could get advantage of the method created by `attr_reader` and instead just call it, this way: 
```suggestion
    items.each do |item|
```